src/Admin/Views/Providers/AddExistingOrganization.cshtml

```diff
@@ -0,0 +1,97 @@
+@using Bit.SharedWeb.Utilities
+@using Microsoft.AspNetCore.Mvc.TagHelpers
+@model OrganizationSearchViewModel
+
+@{
+    ViewData["Title"] = "Add Existing Organization";
+    var providerId = ViewContext.RouteData.Values["id"];
+}
+
+<h1>Add Existing Organization</h1>
+<div class="row mb-2">
+    <div class="col">
+        <form class="form-inline mb-2" method="get" asp-route-id="@providerId">
+            <label class="sr-only" asp-for="OrganizationName">@Html.DisplayNameFor(m => m.OrganizationName)</label>
+            <input type="text" class="form-control mb-2 mr-2 flex-fill" placeholder="@Html.DisplayNameFor(m => m.OrganizationName)" asp-for="OrganizationName" name="name">
+            <label class="sr-only" asp-for="OrganizationOwnerEmail">@Html.DisplayNameFor(m => m.OrganizationOwnerEmail)</label>
```

Comment:
`asp-for` inserts the display name for you, you don't need the `@Html.DisplayNameFor` helpers in the labels.